src/main/java/uk/gov/companieshouse/api/accounts/repository/smallfull/OffBalanceSheetArrangementsRepository.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.api.accounts.repository.smallfull;
+
+import org.springframework.stereotype.Repository;
+import uk.gov.companieshouse.api.accounts.enumeration.AccountingNoteType;
+import uk.gov.companieshouse.api.accounts.model.entity.smallfull.notes.offbalancesheetarrangements.OffBalanceSheetArrangementsEntity;
+import uk.gov.companieshouse.api.accounts.repository.AccountsNoteRepository;
+
+@Repository
+public interface OffBalanceSheetArrangementsRepository  extends AccountsNoteRepository<OffBalanceSheetArrangementsEntity> {
```

Comment:
Sorry I can't find where this interface is being used ?

Reply:
It just inherits default implementations from it's `super`, we've done the same elsewhere where we've extended `MongoRepository`

Reply:
And it's made available by the `NoteRepositoryFactory`